root/static/scripts/edit/URLCleanup.js

```diff
@@ -272,6 +275,9 @@ const CLEANUPS = {
         url = m[1] + encodeURIComponent(decodeURIComponent(m[2])).replace(/%20/g, "_").replace(/%24/g, "$").replace(/%2C/g, ",").replace(/%2F/g, "/").replace(/%3A/g, ":").replace(/%3B/g, ";").replace(/%40/g, "@") + m[3];
       }
       return url;
+    },
+    validate: function (url, id) {
+      return /^https:\/\/[a-z]+\.wikipedia\.org\/wiki\//.test(url);
```

Comment:
This could be tighten by adding the condition `_.includes(CLEANUPS.wikipedia.type, id)` so to discard Wikipedia URLs for other link types, see `amazon` and `youtube` for examples.